core/services/cable_guy/api/manager.py

```diff
@@ -44,40 +68,32 @@ def save(self) -> None:
             print("Configuration is empty, aborting.")
             return
 
-        for item in self.result:
-            item.pop("info")
-        self.settings.save(self.result)
+        result = [dict(filter(lambda field: field[0] != "info", device.__dict__.items())) for device in self.result]
```

Comment:
I'm not sure if this is more readable than the original version.

Reply:
It's not, but I've fixed that already.

It was not a matter of readability in fact. I had to change that because now the self.result stores EthernetInterface instances, not dictionaries, so I have to parse.